### PR TITLE
Rest vit

### PR DESCRIPTION
**Pull request by mushkin-v**

### CODE REVIEW

File: src/AppBundle/Controller/EmployeesController.php [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+{
+    /**
+     * @ApiDoc(
+     *  resource=true,
+     *  description="Returns a collection of Employees",
+     *  statusCodes={
+     *      200="Returned when successful",
+     *      404="Returned when the entities with given limit and offset are not found",
+     *  },
+     *  output = "array<AppBundle\Entity\Employee>"
```
Comment (spolischook) [inline src/AppBundle/Controller/EmployeesController.php]: must be employeesResponse

File: src/AppBundle/Controller/EmployeesController.php [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+     * resource=true,
+     *  description="Returns an Employee by slug",
+     *  statusCodes={
+     *      200="Returned when successful",
+     *      404="Returned when the entity is not found",
+     *  },
+     *  parameters={
+     *      {"name"="Slug", "dataType"="string", "required"=true, "description"="Employee slug"}
+     *  },
+     *  output = "array<AppBundle\Entity\Employee>"
```
Comment (spolischook) [inline src/AppBundle/Controller/EmployeesController.php]: without array<>

File: src/AppBundle/Controller/EmployeesController.php [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+     * resource=true,
+     *  description="Returns an Employee by slug and his roles",
+     *  statusCodes={
+     *      200="Returned when successful",
+     *      404="Returned when the entity is not found",
+     *  },
+     *  parameters={
+     *      {"name"="Slug", "dataType"="string", "required"=true, "description"="Employee slug"}
+     *  },
+     *  output = "array<AppBundle\Entity\Employee>"
```
Comment (spolischook) [inline src/AppBundle/Controller/EmployeesController.php]: array<AppBundle\Entity\Role>

File: src/AppBundle/Controller/PerformanceEventsController.php [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+{
+    /**
+     * @ApiDoc(
+     * resource=true,
+     *  description="Returns a collection of PerformanceEvents",
+     *  statusCodes={
+     *      200="Returned when successful",
+     *      404="Returned when the entities with given limit and offset are not found",
+     * },
+     *  output = "array<AppBundle\Entity\PerformanceEvent>"
```
Comment (spolischook) [inline src/AppBundle/Controller/PerformanceEventsController.php]: PerformanceEventsResponse

File: src/AppBundle/Controller/EmployeesController.php (hunk 3 above)
Comment (mushkin-v) [inline src/AppBundle/Controller/EmployeesController.php]: Array? What you want to say? I dont understand?

File: src/AppBundle/Controller/PerformanceEventsController.php [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+     * resource=true,
+     *  description="Returns one PerformanceEvent by Id",
+     *  statusCodes={
+     *      200="Returned when successful",
+     *      404="Returned when the entity is not found",
+     *  },
+     *  parameters={
+     *      {"name"="Id", "dataType"="string", "required"=true, "description"="PerformanceEvent Id"}
+     *  },
+     *  output = "array<AppBundle\Entity\PerformanceEvent>"
```
Comment (spolischook) [inline src/AppBundle/Controller/PerformanceEventsController.php]: not array

File: src/AppBundle/Controller/EmployeesController.php (hunk 3 above)
Comment (mushkin-v) [inline src/AppBundle/Controller/EmployeesController.php]: oooh! ok))

File: src/AppBundle/Entity/PerformanceEventRepository.php [hunk 6]
```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\EntityRepository;
+
+class PerformanceEventRepository extends EntityRepository
+{
+    public function findByDateRangeEmployees(\DateTime $fromDate, \DateTime $toDate)
```
Comment (spolischook) [inline src/AppBundle/Entity/PerformanceEventRepository.php]: findByDateRange

File: src/AppBundle/Model/EmployeesResponse.php [hunk 7]
```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace AppBundle\Model;
+
+class EmployeesResponse
```
Comment (spolischook) [inline src/AppBundle/Model/EmployeesResponse.php]: Add serialization annotations to this model

File: src/AppBundle/Model/PerformanceEventsResponse.php [hunk 8]
```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace AppBundle\Model;
+
+class PerformanceEventsResponse
```
Comment (spolischook) [inline src/AppBundle/Model/PerformanceEventsResponse.php]: Add serialization annotations to this model

File: src/AppBundle/Model/PerformancesResponse.php [hunk 9]
```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace AppBundle\Model;
+
+class PerformancesResponse
```
Comment (spolischook) [inline src/AppBundle/Model/PerformancesResponse.php]: Add serialization annotations to this model

**Pull request merged**